### PR TITLE
[OTA-2700] Adding another registered user to user's current organization

### PR DESCRIPTION
**Pull request by piotrcichonmee**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+import { Form } from 'formsy-antd';
+import { observer, inject } from 'mobx-react';
+import PropTypes from 'prop-types';
+import isEmail from 'validator/lib/isEmail';
+import { SearchBar } from '../../partials';
+
+@inject('stores')
+@observer
+class ProfileOrganization extends Component {
+  static propTypes = {
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx]: Please move `propTypes` outside of class if possible.

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 2]
```diff
@@ -65,6 +69,19 @@ export default class UserStore {
     this.userOrganizationNamespace = userOrganization.namespace;
   };
 
+  getOrganizationUsers = async () => {
+    const response = await axios.get(API_USER_ORGANIZATIONS_GET_USERS);
+    const { data } = await response;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: Why `await` here?

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 3]
```diff
@@ -65,6 +69,19 @@ export default class UserStore {
     this.userOrganizationNamespace = userOrganization.namespace;
   };
 
+  getOrganizationUsers = async () => {
+    const response = await axios.get(API_USER_ORGANIZATIONS_GET_USERS);
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: Add some kind of error handling

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 4]
```diff
@@ -65,6 +69,19 @@ export default class UserStore {
...
+    const { data } = await response;
+    this.userOrganizationUsers = data;
+  };
+
+  addUserToOrganization = async (email) => {
+    const response = await axios.post(API_USER_ORGANIZATIONS_ADD_USER, { email });
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: Add some kind of error handling

File: ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+                  value={userEmail}
+                />
+              </Form>
+              <Button
+                htmlType="button"
+                className="ant-btn ant-btn-add-registered-user"
+                disabled={!userEmailValid}
+                id="button-add-registered-user"
+                onClick={(event) => {
+                  event.preventDefault();
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx]: Why not just move it to the called function, to keep the `render()` method cleaner?

File: ota-plus-web/app/reactapp/src/stores/UserStore.js (hunk 2 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/stores/UserStore.js]: Done.

File: ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx (hunk 5 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx]: Done

File: ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx]: There is no possible to proper define prop types for userStore. It should be required but at creating static instance of class ProfileOrganization userStore object is undefined.

File: ota-plus-web/app/reactapp/src/stores/UserStore.js (hunk 3 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/stores/UserStore.js]: Done

File: ota-plus-web/app/reactapp/src/stores/UserStore.js (hunk 4 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/stores/UserStore.js]: Done

**Pull request merged**